app/src/components/v-list/v-list-item.vue

```diff
@@ -99,6 +97,24 @@ export default defineComponent({
 			return 'li';
 		});
 
+		const additionalProps = computed(() => {
+			if (props.to) {
+				return {
+					to: props.to,
+				};
+			}
+
+			if (component.value === 'a') {
+				return {
+					href: props.href,
+					target: '_blank',
+					rel: 'noopener noreferrer',
```

Comment:
Unlike v-button, technically `rel` didn't exist before or get passed in this component, but I added it here anyway. This shouldn't affect anything right? 🤔 Just in case `rel` was omitted intentionally for a particular reason I'm not aware of, if any.

Reply:
No particular reason it was omitted!